packages/webapp/src/components/Expandable/ExpandableItem.jsx

```diff
@@ -52,31 +54,35 @@ export default function ExpandableItem({
   return (
     <div className={clsx(classes.container, isExpanded && classes.expandedContainer)}>
       <div
-        className={clsx(
-          styles.mainContentWithIcon,
-          !iconClickOnly && styles.clickable,
-          classes.mainContentWithIcon,
-          isExpanded && classes.expandedMainContentWithIcon,
-        )}
+        className={clsx(classes.alwaysVisibleContent, !iconClickOnly && styles.clickable)}
         onClick={onElementClick}
-        aria-controls={id}
-        aria-expanded={isExpanded}
       >
         <div
           className={clsx(
-            styles.mainContentWrapper,
-            classes.mainContentWrapper,
-            leftCollapseIcon && styles.leftCollapse,
+            styles.mainContentWithIcon,
+            classes.mainContentWithIcon,
+            isExpanded && classes.expandedMainContentWithIcon,
           )}
+          aria-controls={id}
+          aria-expanded={isExpanded}
         >
-          {mainContent}
-        </div>
-        <div
-          onClick={onIconClick}
-          className={clsx(styles.iconWrapper, iconClickOnly && styles.clickable, classes.icon)}
-        >
-          {icons[isExpanded ? 'up' : 'down']}
+          <div
+            className={clsx(
+              styles.mainContentWrapper,
+              classes.mainContentWrapper,
+              leftCollapseIcon && styles.leftCollapse,
+            )}
+          >
+            {mainContent}
+          </div>
+          <div
+            onClick={onIconClick}
+            className={clsx(styles.iconWrapper, iconClickOnly && styles.clickable, classes.icon)}
+          >
+            {icons[isExpanded ? 'up' : 'down']}
+          </div>
         </div>
+        {pillBody && leftCollapseIcon && <Pill body={pillBody} className={styles.pill} />}
```

Comment:
It was a real pain to get styles right... I really tried to avoid editing expandable item.... :( 

I have added stories and moved styles to expandable item. But now I am thinking it would be better served by offering variants...

Reply:
Thank you for the Storybook! ❤️ 

Yeah, I think ultimately the variant we want would maybe just handle the two line wrapping + centring without being particular that the wrapping bit has to be a pill, but I think it's fine to leave like this until we know in what context (if any) we'll see it again.

Reply:
Thanks for the reminder!